scipy/stats/_stats_py.py

```diff
@@ -1606,23 +1608,29 @@ def skewtest(a, axis=0, nan_policy='propagate', alternative='two-sided'):
     agree fairly closely, even for our small sample.
 
     """
+    xp = array_namespace(a)
+    a, axis = _chk_asarray(a, axis, xp=xp)
```

Comment:
Fine to add this since the decorator is only doing this stuff for NumPy arrays (and I don't want to modify it at this time).